bootstrap/mac.sh

```diff
@@ -6,3 +6,13 @@ fi
 
 brew install augeas
 brew install dialog
+
+if ! hash pip 2>/dev/null; then
+    echo "pip Not Installed\nInstalling python from Homebrew..."
+    brew install python
```

Comment:
Doesn't `virtualenv` install `pip` automatically? I.e. is the `brew install python` necessary? We don't need global `pip`, we just care about `./venv/bin/pip`. Actually, I'd prefer not to have global `pip` as this might end up in users typing `sudo pip`.